mythril/analysis/solver.py

```diff
@@ -83,12 +83,13 @@ def get_transaction_sequence(global_state, constraints):
             transactions.append(transaction)
             # Constrain calldatasize
             max_calldatasize = 5000
-            if max_calldatasize != None:
+            if max_calldatasize is not None:
```

Comment:
this conditional is useless anyway right?

Reply:
Yep, didn't notice that.